### ISSUE TITLE
LENGTH ERROR on ]link.create

### ISSUE THREAD

**Issue by bpbecker** [Dyalog/link#118]
1. Clone the Dyalog/DUI repository (let's say to /git/dui/)
2. ]link.create # /git/dui

Results in a LENGTH ERROR

**Comment by abrudz** [comment]
The problem is in FixFiles:
```apl
             nss←U.DotSlash¨paths/⍨~'.'∊¨paths            ⍝ transform simple dirs into nss
             nss←StripCaseCode nss
```
should be something like
```apl
             paths/⍨←~'.'∊¨paths
             nss←U.DotSlash¨paths
             nss←StripCaseCode nss
```

**Comment by nicolas-dyalog** [comment]
Fixed in Link v2.1
